tests/MongoDB.Driver.Core.Tests/Core/Clusters/DnsClientWrapperTests.cs

```diff
@@ -26,38 +27,28 @@ namespace MongoDB.Driver.Core.Clusters
     public class DnsClientWrapperTests
     {
         [Fact]
-        public void constructor_should_initialize_instance()
+        public void Instance_should_return_the_same_instance_each_time()
         {
-            var subject = new DnsClientWrapper();
-
-            subject.LookupClient.Should().NotBeNull();
-        }
```

Comment:
Keep this test but refactor to:

    [Fact]
    public void constructor_should_initialize_instance()
    {
        var subject = CreateSubject();

        subject._lookupClient().Should().NotBeNull();
    }

Reply:
done